### PR TITLE
Edit delete post

### PR DESCRIPTION
**Pull request by partha8**

### CODE REVIEW

File: src/components/DropDown/DropDown.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import { BsThreeDots } from "react-icons/bs";
+import { useState } from "react";
+import { useClickOutside } from "../../hooks/useClickOutside";
+import { useAppDispatch, useAppSelector } from "../../app/hooks";
+import { Posts } from "../../types/posts.types";
+import {
+  deletePost,
+  setEditPost,
+  setPostModal,
+} from "../../features/postsSlice";
```
Comment (Dev-AkshatGupta) [inline src/components/DropDown/DropDown.tsx:11]: Suggestion make a jsoncongif for the imports

File: src/components/DropDown/DropDown.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+export const DropDown = (prop: Posts) => {
+  const [dropdown, setDropdown] = useState(false);
+  const { id } = useAppSelector((store) => store?.auth);
```
Comment (Dev-AkshatGupta) [inline src/components/DropDown/DropDown.tsx:15]: I don't think there is any need of optional chaining in this

**Review by Dev-AkshatGupta — APPROVED** [review]
Great work and liked the error handling the most.
You can merge this good work

File: src/components/DropDown/DropDown.tsx (hunk 1 above)
Comment (partha8) [reply src/components/DropDown/DropDown.tsx:11]: yup, will do!

**Pull request merged**